Periodics/PeriodicsLibrary/Edition.cs

```diff
@@ -0,0 +1,28 @@
+namespace PeriodicsLibrary
+
+{
+    public abstract class Edition
+    {
+        public string Name { get; set; }
+        public int Year { get; set; }
+        public EditionType Type { get; set; }
+        public abstract string GetInfo();
+        public string GetEditionTypeName()
+        {
+            switch (Type)
+            {
+                case EditionType.Almanac:
+                    return "Альманах";
+                case EditionType.Catalogue:
+                    return "Каталог";
+                case EditionType.Magazine:
+                    return "Журнал";
+                case EditionType.Newspaper:
+                    return "Газета";
+                case EditionType.Other:
+                    return "Другое";
+            }
+            return "Нет типа";
+        }
+    }
```

Comment:
В этой реализации я бы отказался от `switch`  и реализовал метод `GetEditionTypeName()` через словарь. 
```suggestion
    public abstract class Edition
    { 
        public string Name { get; set; }
        public int Year { get; set; }
        public EditionType Type { get; set; }
        public abstract string GetInfo();
        
       protected static Dictionary<EditionType, string> TypeName = new Dictionary<EditionType, string>
        {
            {EditionType.Almanac, "Альманах"},
            {EditionType.Catalogue, "Каталог"},
            {EditionType.Magazine, "Журнал"},
            {EditionType.Newspaper, "Газета"},
            {EditionType.Other, "Другое"}
        };

        public string GetEditionTypeName() => TypeName[Type];
    }
```